MmSupervisorPkg/Core/aux_config.spam.cfg

```diff
@@ -1 +1,437 @@
 autogen = true
+
+[[rule]]
+symbol = "gcSmiInitGdtr"
+field = "Base"
+validation.type = "self"
+validation.reference = "NullSeg"
+
+[[rule]]
+symbol = "gcSmiInitGdtr"
+field = "Limit"
+validation.type = "non zero"
+
+[[rule]]
+symbol = "gPatchSmmCr3"
+validation.type = "non zero"
+offset = -4
+size = 4
+
+[[rule]]
+symbol = "gPatchSmmCr4"
+validation.type = "non zero"
+offset = -4
+size = 4
+
+[[rule]]
+symbol = "gPatchSmmCr0"
+validation.type = "content"
+validation.content = [0x33, 0x00, 0x00, 0x80]
+offset = -4
+size = 4
+
+[[rule]]
+symbol = "@PatchLongModeOffset"
+validation.reference = "@LongMode"
+validation.type = "self"
+offset = -6
+size = 4
+
+[[rule]]
+symbol = "gPatchSmmInitStack"
+validation.type = "non zero"
+offset = -8
+size = 8
+
+[[rule]]
+symbol = "@L1"
+validation.type = "self"
+validation.reference = "SmmStartup"
+size = 8
+
+[[rule]]
+symbol = "gPatchRebasedFlagAddr32"
+validation.type = "non zero"
+offset = -4
+size = 4
+
+[[rule]]
+symbol = "gPatchSmmRelocationOriginalAddressPtr32"
+validation.type = "self"
+validation.reference = "mSmmRelocationOriginalAddress"
+offset = -4
+size = 4
+
+# TODO: Really should not do this. We need to update
+# the flow to not patch these values.
+[[rule]]
+symbol = "gPatchSmbase"
+validation.type = "non zero"
+offset = -4
+size = 4
+
+[[rule]]
+symbol = "gPatchSmiStack"
+validation.type = "non zero"
+offset = -4
+size = 4
+
+[[rule]]
+symbol = "gPatchSmiCr3"
+validation.type = "non zero"
+offset = -4
+size = 4
+
+[[rule]]
+symbol = "gPatchMsrIa32MiscEnableSupported"
+validation.type = "non zero"
+offset = -4
+size = 4
+
+[[rule]]
+symbol = "SmiHandlerIdtrAbsAddr"
+validation.type = "self"
+validation.reference = "gSmiHandlerIdtr"
+size = 8
+offset = -8
+
+[[rule]]
+symbol = "mPatchCetSupported"
+validation.type = "content"
+validation.content = [0x00]
+offset = -1
+size = 1
+
+[[rule]]
+symbol = "CpuSmmDebugEntryAbsAddr"
+validation.type = "self"
+validation.reference = "CpuSmmDebugEntry"
+size = 8
+offset = -8
+
+[[rule]]
+symbol = "SmiRendezvousAbsAddr"
+validation.type = "self"
+validation.reference = "SmiRendezvous"
+size = 8
+offset = -8
+
+[[rule]]
+symbol = "CpuSmmDebugExitAbsAddr"
+validation.type = "self"
+validation.reference = "CpuSmmDebugExit"
+size = 8
+offset = -8
+
+[[rule]]
+symbol = "mCetSupportedAbsAddr"
+validation.type = "self"
+validation.reference = "mCetSupported"
+size = 8
+offset = -8
+
+[[rule]]
+symbol = "mXdSupportedAbsAddr"
+validation.type = "self"
+validation.reference = "mXdSupported"
+size = 8
+offset = -8
+
+[[rule]]
+symbol = "gcSmiIdtr"

```

Comment:
Is there any functionality missing from the tool that is making you need to specify the size here?

Reply:
This is an odd case when i initially tested it, where the size is 0. Let me revamp with the new version of tool.

Reply:
Hmm, leaving the size to be automatically determined (Base or Limit) is causing the tool to fail somehow.

Reply:
Fixed with the latest change from https://github.com/microsoft/mu_feature_mm_supv/pull/250